src/components_redux/Input.js

```diff
@@ -29,7 +34,7 @@ class Input extends Component {
   handleChange(i, event) {
```

Comment:
you still have the other handleChange function in this component.
As discussed before you should better use `handleChange  = () => {}` construct to avoid binding.

Reply:
I will change this.